packages/alfa-rectangle/src/rectangle.ts

```diff
@@ -196,6 +196,70 @@ export class Rectangle
     );
   }
 
+  /**
+   * @remarks
+   * Checks if the rectangle intersects a given circle.
```

Comment:
```suggestion
   * Checks if the rectangle intersects a given circle.
   *
   * @remarks
```
(so that the "title" of the comment is what the function does)